database/migrations/20200805131503_update_candle_interval_type/postgres.sql

```diff
@@ -0,0 +1,35 @@
+-- +goose Up
+-- SQL in this section is executed when the migration is applied.
+UPDATE candle SET interval = replace (candle.interval, '15s', '15');
```

Comment:
While its nice that you're offering migration support for people in this PR, its definitely not required! You could always just update the original to the final design and we can just do a "down", "up" command. Anyway, its still appreciated

Reply:
Yeah if at any point i write code to match a current database I like to include a new migration to make changes to it as changes were pushed and people tested with it being a string wanted to include a migration